src/glue/resources/table_columns.yaml

```diff
@@ -0,0 +1,461 @@
+tables:
+  EnrolledParticipants:
```

Comment:
Every table needs at least one of `export_start_date` or `export_end_date` fields. If only a single date is included in the file name (as it is for the EnrolledParticipants data type), then only `export_end_date` is needed. Otherwise both fields are needed.

Reply:
Got it! I think I got confused between seeing if the datatypes themselves have a start date or end date variable in their schemas